### ISSUE TITLE
Changes to db.json should redeploy participate.whatwg.org

### ISSUE THREAD

**Issue by andreubotella** [whatwg/sg#171]
participate.whatwg.org depends on this repo's `db.json` to show the list of workstreams, but #164 added the `testutil` workstream and it is not yet showing up in that list. Changing `db.json` should probably cause participate.whatwg.org to redeploy, much like it does with whatwg.org.

**Comment by annevk** [comment]
As I changed `db.json` this morning I restarted https://github.com/whatwg/participate.whatwg.org/actions/runs/1255401114 in the hope that it does the right thing.

@foolip would fixing this basically involving duplicating the `curl` call in https://github.com/whatwg/sg/blob/main/.github/workflows/dispatch.yml for this repository? Anything else need to be configured?

**Comment by annevk** [comment]
It looks like that did not do the trick as https://participate.whatwg.org/agreement does not list Web IDL.

**Comment by TimothyGu** [comment]
It looks like it's deployed now?
![image](https://user-images.githubusercontent.com/1538624/136124301-8d9c5d8c-ee9f-47c0-ad72-8f98c8dc159d.png)

**Comment by domenic** [comment]
To fix this we need some way of triggering the DigitalOcean infrastructure to redeploy, as the deploys are currently triggered by them monitoring the commits instead of by GitHub Actions pushing a deploy request or something.

**Comment by foolip** [comment]
We could use the same setup as here:
https://github.com/whatwg/whatwg.org/blob/3320e88db45f8abc75696a8bfcc4b8827ecc9d08/.github/workflows/build.yml#L9-L12

**Comment by domenic** [comment]
I don't think this works? This just runs the GitHub Actions, but the GitHub Actions are not what causes a DigitalOcean deploy; DigitalOcean deploys from their side.

**Comment by annevk** [comment]
So https://spec.whatwg.org/ did end up being updated, but let's reopen this so you and @foolip can verify.

**Comment by domenic** [comment]
Yeah, participate.whatwg.org is the tricky part. That got deployed, but ironically only because of all the commits trying to set up deployment (i.e., DigitalOcean detected https://github.com/whatwg/participate.whatwg.org/commit/5bf6e71cf422a41a767a060c0dc9ecc32f7b764a and thus redeployed, including fetching a new version of db.json).

**Comment by annevk** [comment]
Oh right, we did not need a participate update this round because we did not fiddle with Workstreams. So I guess one way of doing this would be to create a commit somehow which ends up triggering DigitalOcean at some point.

**Comment by domenic** [comment]
You do need a participate update to make https://participate.whatwg.org/agreement-status?user=domenic&repo=fs work, actually.
